woocommerce/wc-api-dev
# Prepare composer compatibility

This prepares wc-api-dev to be used by a composer install:

1. Removes the auto-update as it has been unreliable.
2. Adds composer.json for autoload
3. Add check for existing install to avoid conflicts with installed version of plugin

To Test:

1. Download this version in zip form: https://github.com/woocommerce/wc-api-dev/archive/add/composer-json.zip
2. Delete existing wc-api-dev from test site.
3. Install downloaded version
4. Verify normal operation

The composer functionality will have to be tested from the code that installs it, which will have to come after this is merged.
